src/engine/SCons/Tool/MSCommon/vcTests.py

```diff
@@ -48,7 +48,8 @@
 class VswhereTestCase(unittest.TestCase):
     @staticmethod
     def _createVSWhere(path):
-        os.makedirs(os.path.dirname(path))
+        if not os.path.isdir(os.path.dirname(path)):
+            os.makedirs(os.path.dirname(path))
```

Comment:
since it's all Python 3 now, can also do `exist_ok=True` instead of "asking permission".

Reply:
changing!